outputs/core_pools.json

```diff
@@ -42,8 +43,6 @@
     "0x125bc5a031b2db6733bfa35d914ffa428095978b000200000000000000000514": "ECLP-AUSDC-AUSDT",
     "0x2ce4457acac29da4736ae6f5cd9f583a6b335c270000000000000000000004dc": "sFRAX/4POOL",
     "0x2d6ced12420a9af5a83765a8c48be2afcd1a8feb000000000000000000000500": "cbETH/rETH/wstETH",
-    "0x315dd595e82bdc0c194f3a38a08fde480d7e5d2100020000000000000000056a": "ECLP-wUSDM-GYD",
```

Comment:
Bunch of pools here emptied out over STIP.  If they were still on the list they probably would have enough fees to get bribes.  That being said all the LPs have left, and these rewards directed towards other pools would help them sustain the LPs who remain.  Arbitrum looks fine to me.